Gemfile

```diff
@@ -14,6 +14,7 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'rubocop', '~> 0.37.2', require: false
   gem 'rubocop-rspec', require: false
+  gem 'rake', '< 11.0'
```

Comment:
That's fixed in rubocop 0.38 https://github.com/bbatsov/rubocop/commit/4fc1d719f5444a3fb7258255238dd6df4fd35abe